src/main/java/com/unbxd/hummus/controllers/RequestHandler.java

```diff
@@ -0,0 +1,62 @@
+package com.unbxd.hummus.controllers;
+
+
+import com.unbxd.hummus.execptions.SchemaNotFoundException;
+import com.unbxd.hummus.execptions.ServiceException;
+import com.unbxd.hummus.execptions.ValidationException;
+import com.unbxd.hummus.model.Schema;
+import com.unbxd.hummus.services.SchemaService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api")
+public class RequestHandler {
+
+
+    private SchemaService schemaService;
+
+    @Autowired
+    public RequestHandler(SchemaService schemaService) {
+        this.schemaService = schemaService;
+    }
+
+    @RequestMapping("/")
+    String ping() {
+        return "Hummus!";
+    }
+
+    @RequestMapping(value = "/addSchema",method = RequestMethod.POST)
+    ResponseEntity<String> addSchema(@RequestBody Schema schema) {
+        try {
+            schemaService.addSchema(schema);
+        }
+        catch (ValidationException e) {
```

Comment:
use Error/Exception handler instead of catching the exception and creating response on every error